enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleDescription.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import type { MouseEvent } from "react";
+import { t } from "ttag";
+import { Anchor, Popover, Stack, Text } from "metabase/ui";
+
+export function MetabaseLinksToggleDescription() {
+  const [opened, setOpened] = useState(false);
```

Comment:
I changed this component to be controlled because I want to use `onClick` on the `Anchor` (it doesn't work on uncontrolled version).